src/screens/Settings/components/AdvancedSettings/index.tsx

```diff
@@ -216,7 +216,7 @@ export const AdvancedSettings = ({
             isSuccess: isCopiedToClipboard
           })}
           onClick={() => {
-            clipboard.writeText(JSON.stringify({ ...settingsToSave }))
+            clipboard.writeText(JSON.stringify({ ...settingsToSave }, null, 2))
```

Comment:
added this so the copied settings in the clipboard are pretty printed, easier to read